src/union-fuse/main.c

```diff
@@ -0,0 +1,752 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*-
+ *
+ * Copyright (C) 2015,2016 Colin Walters <walters@verbum.org>
```

Comment:
I have based it all on rofiles-fuse (copied both Makefile-inc.am and main.c) that is why I left the original copyright line.  I added the copyright line to Red Hat, since I've worked on it during my working hours